.github/workflows/test-coverage.yaml

```diff
@@ -15,40 +17,45 @@ jobs:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - uses: r-lib/actions/setup-r@v2
         with:
           use-public-rspm: true
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
+          extra-packages: any::covr, any::xml2
           needs: coverage
 
-      - name: Setup
-        run: |
-          R CMD INSTALL .
-          source shim_package.sh
```

Comment:
This is critically important to run covr on itself, because you get into an infinite recursion issue if you don't rename the package to something else before trying to run covr on it, which is what shim_package.sh does.

Reply:
Oops, I missed that.